### PR TITLE
Fixed filter type ahead

### PR DESCRIPTION
**Pull request by dlabrecq**
Fixed the filter type ahead feature. Also moved type ahead to separate example for clarity.

Example:
https://rawgit.com/dlabrecq/patternfly-ng/type-ahead-dist/dist-demo/

Screen shot:
screen shot 2017-06-22 at 11 43 20 am

### CODE REVIEW

File: src/app/filters/examples/filter-basic-example.component.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    </div>
+  </div>
+  <div class="row padding-bottom-15 padding-top-15">
+    <div class="col-xs-12">
+      <h4>Code</h4>
+      <hr/>
+    </div>
+  </div>
+  <div>
+    <tabset style="border: 1px black;">
```
Comment (AdamJ) [inline src/app/filters/examples/filter-basic-example.component.html]: any specific reason why the inline style here vs a class?

File: src/app/filters/examples/filter-type-ahead-example.component.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    </div>
+  </div>
+  <div class="row padding-bottom-15 padding-top-15">
+    <div class="col-xs-12">
+      <h4>Code</h4>
+      <hr/>
+    </div>
+  </div>
+  <div>
+    <tabset style="border: 1px black;">
```
Comment (AdamJ) [inline src/app/filters/examples/filter-type-ahead-example.component.html]: same question here on inline styles

Comment (dlabrecq) [reply src/app/filters/examples/filter-type-ahead-example.component.html]: This is just the example code. I'm planning to replace that with a style class (for all examples) after all the latest PRs are merged -- it's been merge h*ll getting everything integrated.

Comment (AdamJ) [reply src/app/filters/examples/filter-type-ahead-example.component.html]: ok-sounds good

**Review by AdamJ — APPROVED** [review]

File: src/app/filters/examples/filter-type-ahead-example.component.html (hunk 2 above)
Comment (dlabrecq) [reply src/app/filters/examples/filter-type-ahead-example.component.html]: Created a PR to remove the tab style and use .tab-content from ngx-bootstrap, instead.

See: https://github.com/patternfly/patternfly-ng/pull/34

**Review by dgutride — APPROVED** [review]

**Pull request merged**